spanner/emulator_test.sh

```diff
@@ -24,7 +24,8 @@ export GCLOUD_TESTS_GOLANG_PROJECT_ID=emulator-test-project
 echo "Running the Cloud Spanner emulator: $SPANNER_EMULATOR_HOST";
 
 # Download the emulator
-EMULATOR_VERSION=1.2.0
+# TODO: Find a way to use 'latest' here.
+EMULATOR_VERSION=1.4.0
```

Comment:
isnt there a 1.4.1 version ?

Reply:
You're right. I've bumped the version to 1.4.1.